wolfcrypt/src/random.c

```diff
@@ -2048,6 +2049,19 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
         return ret;
     }
 
+#elif defined(WOLFSSL_WICED)
+    int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
+    {
+        (void)os;
+        if (output == NULL || UINT16_MAX < sz) {
+            return BUFFER_E;
+        }
+
+        wiced_crypto_get_random((void*) output, sz);
```

Comment:
Does `wiced_crypto_get_random()` return a value?  If so, we should be checking it here and returning an error if it fails.